browser/module.go

```diff
@@ -69,3 +76,459 @@ func (*RootModule) NewModuleInstance(vu k6modules.VU) k6modules.Instance {
+		"dblclick":                p.Dblclick,
+		"dispatchEvent":           p.DispatchEvent,
+		"dragAndDrop":             p.DragAndDrop,
+		"emulateMedia":            p.EmulateMedia,
+		"emulateVisionDeficiency": p.EmulateVisionDeficiency,
+		"evaluate":                p.Evaluate,
+		"evaluateHandle":          p.EvaluateHandle,
+		"exposeBinding":           p.ExposeBinding,
+		"exposeFunction":          p.ExposeFunction,
+		"fill":                    p.Fill,
+		"focus":                   p.Focus,
+		"frame":                   p.Frame,
+		"frames": func() *goja.Object {
+			var (
+				mfrs []mapping
+				frs  = p.Frames()
+			)
+			for _, fr := range frs {
+				mfrs = append(mfrs, mapFrame(rt, fr))
+			}
+			return rt.ToValue(mfrs).ToObject(rt)
+		},
+		"getAttribute": p.GetAttribute,
+		"goBack":       p.GoBack,
+		"goForward":    p.GoForward,
+		"goto":         p.Goto,
+		"hover":        p.Hover,
+		"innerHTML":    p.InnerHTML,
+		"innerText":    p.InnerText,
+		"inputValue":   p.InputValue,
+		"isChecked":    p.IsChecked,
+		"isClosed":     p.IsClosed,
+		"isDisabled":   p.IsDisabled,
+		"isEditable":   p.IsEditable,
+		"isEnabled":    p.IsEnabled,
+		"isHidden":     p.IsHidden,
+		"isVisible":    p.IsVisible,
+		"locator":      p.Locator,
+		"mainFrame": func() *goja.Object {
+			mf := mapFrame(rt, p.MainFrame())
+			return rt.ToValue(mf).ToObject(rt)
+		},
+		"opener": p.Opener,
+		"pause":  p.Pause,
+		"pdf":    p.Pdf,
+		"press":  p.Press,
+		"reload": func(opts goja.Value) *goja.Object {
+			r := mapResponse(rt, p.Reload(opts))
+			return rt.ToValue(r).ToObject(rt)
+		},
+		"route":                       p.Route,
+		"screenshot":                  p.Screenshot,
+		"selectOption":                p.SelectOption,
+		"setContent":                  p.SetContent,
+		"setDefaultNavigationTimeout": p.SetDefaultNavigationTimeout,
+		"setDefaultTimeout":           p.SetDefaultTimeout,
+		"setExtraHTTPHeaders":         p.SetExtraHTTPHeaders,
+		"setInputFiles":               p.SetInputFiles,
+		"setViewportSize":             p.SetViewportSize,
+		"tap":                         p.Tap,
+		"textContent":                 p.TextContent,
+		"title":                       p.Title,
+		"type":                        p.Type,
+		"uncheck":                     p.Uncheck,
+		"unroute":                     p.Unroute,
+		"uRL":                         p.URL,
+		"video":                       p.Video,
+		"viewportSize":                p.ViewportSize,
+		"waitForEvent":                p.WaitForEvent,
+		"waitForFunction":             p.WaitForFunction,
+		"waitForLoadState":            p.WaitForLoadState,
+		"waitForNavigation":           p.WaitForNavigation,
+		"waitForRequest":              p.WaitForRequest,
+		"waitForResponse":             p.WaitForResponse,
+		"waitForSelector":             p.WaitForSelector,
+		"waitForTimeout":              p.WaitForTimeout,
+		"workers":                     p.Workers,
+		"query": func(selector string) *goja.Object {
```

Comment:
We don't need `query` and `queryAll`, no?

Reply:
No, we don't :)